moveit_planners/ompl/CMakeLists.txt

```diff
@@ -42,5 +38,12 @@ include_directories(ompl_interface/include
 
 add_subdirectory(ompl_interface)
 
-install(FILES ompl_interface_plugin_description.xml
-  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
+#install(FILES ompl_interface_plugin_description.xml
+#  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
+pluginlib_export_plugin_description_file(moveit_ompl_interface ompl_interface_plugin_description.xml)
+
+ament_export_include_directories(include)
+ament_export_libraries(moveit_ompl_interface)
+ament_export_dependencies(moveit_core OMPL)
```

Comment:
Shouldn't that be `ompl`? Where is this defined?

Reply:
You're probably right. However, I'm not sure how ament handles this since the include/library targets have the prefix `OMPL` which is why also `ament_target_dependencies()` only works with `OMPL`.

Reply:
Also, `OMPL` was used for catkin_package `DEPENDS` as well.